src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -85,23 +85,30 @@ const ProfileHeader = ({
     'medium',
   );
 
+  const classes = {
+    wrapper: [...wrapperClasses, ...wrapperClassesMedium].join(' '),
+    innerWrapper: [
+      ...innerWrapperClasses,
+      ...innerWrapperClassesMedium,
+      'usa-grid',
+      'usa-grid-full',
+    ].join(' '),
+    serviceBadge: [
+      ...serviceBadgeClasses,
+      ...serviceBadgeClassesMedium,
+      'usa-width-one-fourth',
+    ].join(' '),
+    title: [...titleClasses, ...titleClassesMedium].join(' '),
+    fullName: [...fullNameClasses, ...fullNameClassesMedium].join(' '),
+    latestBranch: [...latestBranchClasses, ...latestBranchClassesMedium].join(
+      ' ',
+    ),
+  };
```

Comment:
I love this! I'd still consider moving it to the parent scope outside of the component as constants, or do you prefer it here? From my very limited experience with it, `react-native` usually does that with its style definitions

Reply:
I don't have super strong feelings about this one way or another. I guess the benefits of defining them outside of the component would be that it'd be easier to test those constants if you wanted to and also be able to more easily swap out different style objects. Anyway this is a huge improvement, IMO. Thanks for the suggestion.